package.json

```diff
@@ -109,6 +111,8 @@
     "react-native-modalize": "^2.0.6",
     "react-native-orientation-locker": "^1.2.0",
     "react-native-paper": "^3.11.0",
+    "react-native-permissions": "^2.2.2",
+    "react-native-qrcode-scanner": "^1.4.1",
```

Comment:
No related @types dep?

Reply:
not as I see on the documentation:

https://github.com/moaazsidat/react-native-qrcode-scanner